src/cdk/drag-drop/drag-drop.md

```diff
@@ -128,6 +128,17 @@ to be applied.
 
 <!-- example(cdk-drag-drop-custom-preview) -->
 
+### Customizing the drag preview container
+When a `cdkDrag` element is picked up, it will create a preview element which is appended to
+the end of `<body>`. There may be times when you need to have the preview created elsewhere.

```

Comment:
We should have another sentence here explaining that the element is appended to the body in order to avoid issues with `z-index` and `overflow: hidden`.

Reply:
how's this sound?
```
/**
   *  Target for Drag Preview clone to append into.
   *  `global` will place the clone at the end of `<body>`.
   *  `parent` will place the clone in the parent to the `CdkDropList` .
   *  `ElementRef` or `HTMLElement` will place the clone inside the element.
   *
   *  When `global` (default), the Drag Preview is appended to `<body>` to avoid
   *  potential issues with `z-index` and `overflow: hidden`. By changing the
   *  location you run risk of ancestors styled with `overflow: hidden` hiding part
   *  or all of the Drag Preview, as well as other items styled with a higher
   *  `z-index` overlapping the preview container.
   */
```